openupgrade_scripts/scripts/analytic/17.0.1.2/pre-migration.py

```diff
@@ -22,6 +25,30 @@ def _fill_config_parameter_analytic_project_plan(env):
         [("module", "=", "analytic"), ("name", "=", "analytic_plan_projects")]
     )
     plan_id = imd.res_id
+    if not plan_id and openupgrade.column_exists(
+        env.cr, "res_company", "internal_project_id"
+    ):
+        env.cr.execute(
+            "SELECT internal_project_id "
+            "FROM res_company "
+            "WHERE internal_project_id IS NOT NULL "
+            "LIMIT 1;"
```

Comment:
```suggestion
            """SELECT internal_project_id
            FROM res_company
            WHERE internal_project_id IS NOT NULL
            LIMIT 1
            ORDER BY id"""
```

Reply:
Turns out this column does not exist in DB...